save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/AgentsController.kt

```diff
@@ -67,6 +73,22 @@ class AgentsController {
         dockerService.stopAgents(agentIds)
     }
 
+    /**
+     * @param executionLogs ExecutionLogs
+     */
+    @PostMapping("/executionLogs")
+    fun saveAgentsLog(@RequestBody executionLogs: ExecutionLogs) {
+        val logFile = File(configProperties.agentLogs + File.separator + "${executionLogs.agentId}.log")
```

Comment:
Maybe, we can simply create a dedicated slf4j logger, configure rolling file policy for it and write received events through it, instead of manually configuring everything?

Reply:
Ok, it would be too complicated to configure this for every agent